test/Succeed/ExecAgda.agda

```diff
@@ -9,15 +9,11 @@ open import Agda.Builtin.Unit
 open import Agda.Builtin.Sigma
 open import Agda.Builtin.String
 open import Agda.Builtin.Reflection renaming (bindTC to _>>=_)
-
-postulate
-  execTC : String → List String → String → TC (Σ Nat (λ _ → Σ String (λ _ → String)))
-
-{-# BUILTIN AGDATCMEXEC execTC #-}
+open import Agda.Builtin.Reflection.External
 
 macro
   test : Term → TC ⊤
-  test hole = execTC "agda" ("-v0" ∷ "-i" ∷ "test/Succeed" ∷ "test/Succeed/exec-tc/empty.agda" ∷ []) ""
+  test hole = execTC "agda" ("-v0" ∷ "-i" ∷ "test/Succeed" ∷ "exec-tc/empty.agda" ∷ []) ""
```

Comment:
The test was correct. The `Makefile` is called from `../..` hence the need for the relative path.